pkg/test/integration/testdata/otel-ebpf-profiler-offcpu-cpu.json.bin

```diff
@@ -51420,65 +51198,38 @@
         },
         {
           "address": "0x3246730",
-          "lines": [
-            "libxul.so 0x3246730[]@:0"
-          ],
           "mapping": "0x2995000-0x8ddb000@0x2994000 libxul.so(ed82595fe72c9215b49e7e2cb2f85ea9269c1a61)"
         },
         {
           "address": "0x3246290",
-          "lines": [
```

Comment:
I think we should keep these synthetic frames if we do not symbolize them or fail to symbolize

Consider this flamegraph
![Screenshot 2025-04-17 at 13 37 39](https://github.com/user-attachments/assets/7b877f33-1c8c-4e8d-b4a1-92044652d5fd)

It hides the whole userspace stack. it is at least make harder to debug things for us, and at most may change the shape and look of flamegraph drastically if these frames are skipped in the middle of symbolized frames

Reply:
I think this is happening for somewhat strange behaviour. I'm getting the following mappings:
```
1: 0x0/0x0/0x0   
2: 0x28000/0x1b0000/0x28000 libc.so.6 42c84c92e6f98126b3e2230ebfdead22c235b667 
3: 0x0/0x2000/0x0 linux-vdso.1.so
```
Mapping 1 has ~880 locations, so it's likely the one with userspace data. However, without a valid build id, we skip it; it's not even reaching symbolization. It's strange as we had valid build ids for `pyroscope-ebpf/profiler` before 🤔 

This is for the `pyroscopee-ebpf-otel-dev-001/otel-profiler`

<img width="1630" alt="Screenshot 2025-04-22 at 12 24 33" src="https://github.com/user-attachments/assets/a47416cd-446f-4618-b102-54ea7f926fb3" />

Reply:
I think we still something is broken here and we now do not output the stubs.
I also think is mostly important for profiler developers and me, but not the profiling users. 
I will resolve this, we can address this later